### PR TITLE
dwarfutils 0.3.2

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/dwarfutils.rb [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-  sha256 "f2592e56aa5d8f39869367edc80bfe6529c04a5223219f628b6249b5bdd5f721"
+  url "https://github.com/davea42/libdwarf-code/releases/download/libdwarf-0.3.2/libdwarf-0.3.2.tar.xz"
+  sha256 "a5b150182a1ce18e16b2f4225d3902b3f2b30181d2678f29cbd90b5ee7b067d1"
   license all_of: ["BSD-2-Clause", "LGPL-2.1-or-later", "GPL-2.0-or-later"]
   version_scheme 1
 
-  livecheck do
-    url :homepage
-    regex(%r{href=(?:["']?|.*?/)libdwarf[._-]v?(\d+(?:\.\d+)+)\.t}i)
-  end
```
Comment (chenrui333) [inline Formula/dwarfutils.rb:12]: from https://www.prevanders.net/dwarf.html#releases
> As of release 0.3.1 releases are also available for download from the github libdwarf page

cc @samford

**Review by samford — COMMENTED** [review]
Not sure whether it's preferable to download the `stable` archive from the first-party site or GitHub. If using GitHub is preferable, the `livecheck` block changes look good to me.

I'll let someone more knowledgeable about the `stable` URL change (to GitHub) approve this. For what it's worth, it looks like ~45% of formulae with a github.com `stable` URL use a non-github.com `homepage`, so this doesn't seem to be an uncommon situation. The question is more about whether we would/should switch a `stable` URL to GitHub when the first-party source is otherwise fine.

**Review by nandahkrishna — COMMENTED** [review]
Seems like the change to GitHub might be because https://www.prevanders.net/libdwarf-0.3.2.tar.xz gives a 404 Error. Perhaps due to this note

> As of release 0.3.1 releases are also available for download from the github libdwarf page

and also the unlikelihood of broken links on GitHub Releases, the switch to GitHub URLs seems okay to me.

**Comment by samford** [comment]
> Seems like the change to GitHub might be because https://www.prevanders.net/libdwarf-0.3.2.tar.xz gives a 404 Error.

Ah, good catch! It's strange that the homepage links to a non-existent file but that's a good enough indicator to me to go ahead with switching to GitHub 👍

**Review by samford — APPROVED** [review]

**Comment by chenrui333** [comment]
> Seems like the change to GitHub might be because https://www.prevanders.net/libdwarf-0.3.2.tar.xz gives a 404 Error. Perhaps due to this note
> 
> > As of release 0.3.1 releases are also available for download from the github libdwarf page
> 
> and also the unlikelihood of broken links on GitHub Releases, the switch to GitHub URLs seems okay to me.

yeah, pretty much, also logged an issue about it.
- https://github.com/davea42/libdwarf-code/issues/52

**Pull request closed without merging**